app/ssttpcalculator/BigDecimalUtil.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ssttpcalculator
+
+object BigDecimalUtil {
+
+  def roundDownToNearestTen(value: BigDecimal): BigDecimal = BigDecimal((value.longValue() / 10) * 10)
```

Comment:
Iirc from ancient Java days, `BigDecimal` has methods to support careful/exact rounding of numbers avoiding the need for using the long value there. 
Rarely for Java, it's actually an immutable data type on to which you can chain multiple maths operations in sequence.

Reply:
Nb; The `value.longValue` will truncate any fractional part - is this really what's intended? I can't tell exactly how it's inteded to behave from function name alone.

Reply:
yeah, that is intended. We support values from 32 to 30000 and want to display a number between divided by  another number from range 3 to 12. 


>Iirc from ancient Java days, BigDecimal has methods to support careful/exact rounding of numbers avoiding the need for using the long value there.
Yes, I know that, but these methods sets the precision inside the big decimal. I didn't want to create a decimal with such precision and returning it back, because it was not the purpose of this method. 
I could set the precision and rounding mode and then reset them to back what it was, but decided to follow existing convention.

Reply:
ps. looks like bigdecimail doesn't support rounding to 10 via `setScale` and `roundingMode` thats why so hacky solution